geomstats/sampling/lognormal.py

```diff
@@ -0,0 +1,131 @@
+"""LogNormal Sampler"""
+
+import geomstats.backend as gs
+from geomstats.geometry.euclidean import Euclidean
+from geomstats.geometry.spd_matrices import SPDMatrices
+
+
+class LogNormal:
+    """LogNormal Sampler on manifold of SPD Matrices and Euclidean Spaces.
+
+    (1)For Euclidean Spaces, if X is distributed as Normal(mean, cov), then
+    exp(X) is distributed as LogNormal(mean, cov).
+
+    (2)For SPDMatrices, LogNormal distribution is defined by embedding mean
+    in the Euclidean Space; mean(a SPD matrix) is embedded into
+    Symmetric Matrices Space using Matrix Logarithm, which is then embedded
+    into Euclidean Space. This embedded mean and given cov is used to
+    parametrize Normal Distribution.
+
+    Parameters
+    ----------
+    manifold: Manifold obj, {Euclidean(n), SPDMatrices(n)}
+        Manifold to sample over. Manifold should
+        be of instance of Euclidean or SPDMatrices
+    mean: array-like,
+            shape=[dim] if manifold is Euclidean Space
+            shape=[n, n] if manifold is SPD Manifold
+        Mean of the distribution
+
+    cov: array-like,
+            shape=[dim, dim] if manifold is Euclidean Space
+            shape=[n*(n+1)/2, n*(n+1)/2] if manifold is SPD Manifold
+        Covariance of the Distribution.
+
+    Returns
+    --------
+    samples: array-like, shape[..., dim]
+        Samples from LogNormal distribution
+
+    Examples
+    --------
+    import geomstats.backend as gs
```

Comment:
Nice, very good idea to add an example!! Could you use this syntax? https://github.com/geomstats/geomstats/blob/eb1716ee3d8553123c779c23af65ecac5ec0aa08/geomstats/learning/online_kmeans.py#L144

This way, the example would better render on our documentation website.